distribution/docs/src/main/jdocs/content/_plugins/metacard-validity-marker-contents.adoc

```diff
@@ -0,0 +1,33 @@
+:type: plugin
+:status: published
+:title: Metacard Validity Marker
+:link: _metacard_validity_marker
+:plugintypes: preingest
+:summary: Modifies metacards when created or ingested according to metacard validator services.
+
+The Metacard Validity Marker Pre-Ingest plugin modifies the metacards contained in ``CreateRequest``s and ``UpdateRequest``s.
+
+The plugin runs each metacard in the `CreateRequest` and `UpdateRequest` against each registered `MetacardValidator` service.
+
+[NOTE]
+====
+This plugin can make it seem like ingested products are not successfully ingested.
+If after ingesting, the ingest did not fail and there are no errors in the ingest log, but the expected results do not show up after a query,
+verify either that the ingested data is valid or that the <<catalogFederationStrategy, Catalog Federation Strategy>> is configured to show warnings and/or errors.
```

Comment:
❓ Perhaps this should reference the filter plugin instead, and recommend logging in as a permitted user and querying for invalid metacards. Changing the caching federation strategy could have broader, unintended side effects.